spec/facility_spec.rb

```diff
@@ -1,26 +1,211 @@
-require 'spec_helper'
+# typed: ignore
 
-RSpec.describe Facility do
-  before(:each) do
-    @facility = Facility.new({name: 'DMV Tremont Branch', address: '2855 Tremont Place Suite 118 Denver CO 80205', phone: '(720) 865-4600'})
+require "spec_helper"
+
+RSpec.describe Facility do # rubocop:disable Metrics/BlockLength
+  before(:each) do # rubocop:disable Metrics/BlockLength
+    @facility1 = Facility.new({
+      name: "DMV Tremont Branch",
+      address: "2855 Tremont Place Suite 118 Denver CO 80205",
+      phone: "(720) 865-4600"
+    })
+    @facility2 = Facility.new({
+      name: "DMV Northeast Branch",
+      address: "4685 Peoria Street Suite 101 Denver CO 80239",
+      phone: "(720) 865-4600"
+    })
+    @cruz = Vehicle.new({
+      vin: "123456789abcdefgh",
+      year: 2012,
+      make: "Chevrolet",
+      model: "Cruz",
+      engine: :ice
+    })
+    @bolt = Vehicle.new({
+      vin: "987654321abcdefgh",
+      year: 2019,
+      make: "Chevrolet",
+      model: "Bolt",
+      engine: :ev
+    })
+    @camaro = Vehicle.new({
+      vin: "1a2b3c4d5e6f",
+      year: 1969,
+      make: "Chevrolet",
+      model: "Camaro",
+      engine: :ice
+    })
+    @registrant = Registrant.new({
+      name: "Bruce",
+      age: 18,
+      permit: true
+    })
+  end
+  before(:all) do # rubocop:disable Metrics/BlockLength
+    @co_office = DmvDataService.new.co_dmv_office_locations[0]
+    @ny_office = DmvDataService.new.ny_dmv_office_locations[0]
+    @mo_office = DmvDataService.new.mo_dmv_office_locations[0]
+
+    @co_facility = Facility.new({
```

Comment:
This code should actually be living in a method within the code, rather than the test. Maybe in some kind of FacilityFactory that is taking in the raw API data and converting that raw data into Facility objects. Then you'd be able to write tests to make sure the method itself is working. Here, it's almost like you're testing your test.